react/src/hooks/utils.tsx

```diff
@@ -1,5 +1,7 @@
 // Function patterns that commonly occur in this set of hooks.
 
+import { Query as MTQuery, QueryResult as MTQueryResult } from "material-table";
```

Comment:
Does the name conflict with a different import? The default name is used in a different file above.